docs/src/terminology.md

```diff
@@ -36,13 +48,17 @@ The first [validator](terminology.md#validator) to produce a [block](terminology
 
 Smallest encrypted chunk of ledger, an encrypted ledger segment would be made of many CBC blocks. `ledger_segment_size / cbc_block_size` to be exact.
 
+## checkpoint
+
+If there are multiple active forks within the blockstore, each point in which two forks meet would be a checkpoint. When a checkpoint is older than the root, it can be pruned from memory.
```

Comment:
Where are you seeing this? We shouldn't be using the term "checkpoint". If we do, it should be synonymous with "snapshot".

Reply:
https://docs.solana.com/cluster/managing-forks

Reply:
Okay, thanks. Yes, that's a little different than a snapshot.  In the codebase, we call it a "frozen bank", but in that high-level document, calling it a checkpoint seems fine.  How about a definition of, "The ledger state at a given slot height."?